src/docs/FgImageContainerDocs.vue

```diff
@@ -1,40 +1,40 @@
 <template>
   <div>
-    <divider/>
+    <fg-divider/>
 
-    <header-large>
-      IMAGE CONTAINER
-    </header-large>
+    <fg-header-large>
+      FG IMAGE CONTAINER
```

Comment:
I would make this header match the related label used in the side menu and therefore would drop the prefix:
```suggestion
      IMAGE CONTAINER
```